codegen/winterfell/src/air/boundary_constraints.rs

```diff
@@ -128,7 +128,7 @@ fn add_aux_trace_assertions(func_body: &mut codegen::Function, ir: &Air) {
 
                     func_body.line(assertion);
                 }
-                air_ir::BusBoundary::Null => {}
+                air_ir::BusBoundary::Null | air_ir::BusBoundary::Unconstrained => {}
```

Comment:
We don't do anything for `Null` constraints here because they a part of the algebraic graph, right? If so, I'd add a comment to explain this.